riff-raff/app/assets/stylesheets/graph.less

```diff
@@ -40,14 +40,14 @@
   margin-left: 20px;
   padding: 2px 0;
   background-color: transparent;
-  color: black;
+  color: #ffffff;
```

Comment:
`black` didn't provide enough contrast, so move to `ffffff` aka white

**before**
![image](https://user-images.githubusercontent.com/836140/68613065-d48fb180-04b5-11ea-85af-ec1c63d22405.png)

**after**
![image](https://user-images.githubusercontent.com/836140/68613036-c346a500-04b5-11ea-97fb-05c3a5c7b0b9.png)